sdks/python/apache_beam/runners/worker/opcounters.py

```diff
@@ -42,6 +43,58 @@ def value(self):
     return self._value
 
 
+class TransformIoCounter(object):
```

Comment:
<!--new_thread; commit:6e2c396b4cdcce20867c583cba65c8664ff79996; resolved:0-->
Can you add documentation ? Also is this a user-facing interface ? If so this should be discussed more broadly.

Reply:
This won't be a user-facing interface. It's meant to be used by the IO infrastructure classes. I'll add documentation in a bit.